dspace-api/src/main/java/org/dspace/app/mediafilter/MediaFilterServiceImpl.java

```diff
@@ -382,10 +398,21 @@ public boolean processBitstream(Context context, Item item, Bitstream source, Fo
 
         Bitstream b = bitstreamService.create(context, targetBundle, destStream);
 
+        // The input stream was stored in the appropriate asset store, close it to avoid socket leaks.
+        if (destStream != null)
+        {
+            try {
+                destStream.close();
+            } catch (IOException ex)
+            {
+                // we cannot recover from not being able to close the input stream, so ignore the exception.
+            }
+        }
```

Comment:
The above if-block can be replaced by org.apache.commons.io.IOUtils#closeQuietly(java.io.InputStream)